scripts/deploy-one.js

```diff
@@ -1,92 +1,105 @@
-const fs   = require("fs");
-const Web3 = require("web3");
+const fs = require('fs');
+const path = require('path');
+const Web3 = require('web3');
 
-const NODE_ADDRESS  = process.argv[2];
-const PRIVATE_KEY   = process.argv[3];
+const NODE_ADDRESS = process.argv[2];
+const PRIVATE_KEY = process.argv[3];
 const CONTRACT_NAME = process.argv[4];
 const CONTRACT_ARGS = process.argv.slice(5);
 
-async function scan(message) {
+async function scan (message) {
     process.stdout.write(message);
-    return await new Promise(function(resolve, reject) {
+    return await new Promise(function (resolve, reject) {

```

Comment:
need to go over the linter rules but as an example, we prefer arrow style and no space between `function` and its args

Reply:
This is a leftover of the previous version of the script, but I'll quickly fix it everywhere. If you see additional style violation, please let me know and I'll reformat.